### DISCUSSION TITLE
Cannot find module '@docusaurus/theme-common/internal' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by pincman** [facebook/docusaurus#7914] [Q&A]
work fine , but vscode show error `Cannot find module '@docusaurus/theme-common/internal' or its corresponding type declarations.`

**Comment by slorber** [comment]
It does exist, but is using a quite new Node.js package "exports" field. Maybe your VSCode version is not up to date and does not support it yet?

**Comment by Josh-Cena** [comment]
You need to add `"module": "NodeNext"` to your tsconfig. It's not enabled in the shared config yet—see https://github.com/tsconfig/bases/pull/122
